_includes/head.html

```diff
@@ -3,7 +3,7 @@
 <head>
     <link rel="shortcut icon" href="images/club_logo.png" type="image/x-icon"/>
     <title>FRC 6321 Scouting</title>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" name='viewport' content="width=device-width, initial-scale=1.0">
```

Comment:
seperate meta lines, first says what character mapping you're using, the second sais what to set the viewport to.

```html
<meta charset="UTF-8" />
<meta name='viewport' content="width=device-width, initial-scale=1.0" />
```